cli/internal/helm/charts/edgeless/constellation-services/charts/ccm/values.schema.json

```diff
@@ -0,0 +1,108 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "properties": {
+        "csp": {
+            "description": "CSP to which the chart is deployed.",
+            "enum": ["Azure", "GCP", "AWS", "QEMU"]
+        },
+        "subnetworkCIDR": {
+            "description": "CIDR for the subnetwork of the cluster",
+            "type": "string",
+            "examples": ["192.0.2.0/24"],
+            "pattern": "[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}/[0-9][0-9]"
+        },
+        "GCP": {
+            "description": "Config values required for deployment on GCP",
+            "type": "object",
+            "properties": {
+                "image": {
+                    "description": "Container image to use for the spawned pods.",
+                    "type": "string",
+                    "examples": ["ghcr.io/edgelesssys/cloud-provider-gcp:latest"],
+                    "pattern": "ghcr.io/edgelesssys/cloud-provider-gcp:.+"
```

Comment:
While the idea is nice, this prevents users from using their own cloud provider image, which might be desirable.
Same for AWS and Azure